styx-common/src/test/java/com/spotify/styx/model/data/deprecated/WorkflowInstanceExecutionDataTest.java

```diff
@@ -136,41 +68,41 @@ public void shouldDeserializeExecutionData() throws Exception {
         + "\"workflow_instance\":{"
           + "\"workflow_id\":{"
             + "\"component_id\":\"component1\","
-            + "\"id\":\"endpoint1\""
+            + "\"endpoint_id\":\"endpoint1\""
```

Comment:
Most tests in this deprecated class seem to be the same of the non deprecated one. Shouldn't this class test the deprecated API (that uses `endpoint_id`)?